#459 Adjustment Sale Order View

nakometal opened the pull request:
## Description

Moved fields in Sale Order because of the fields always need to input.  
`date_order` has not been changed yet.

### fields

* warehouse_id
* expected_date
* commitment_date
* user_id

## Module

**sale_view_adj_hls**

nakometal committed:
[ADD] sale_view_adj_hls

nakometal committed:
[ADD] invisible fields

nakometal committed:
[ADD] header fields

yostashiro commented on sale_view_adj_hls/__manifest__.py:
```diff
@@ -0,0 +1,22 @@
+    "application": False,
+    "installable": True,
+    "depends": [
+        "sale"
```
`sale_stock` instead of `sale`.

yostashiro commented on sale_view_adj_hls/views/sale_order_views.xml:
```diff
@@ -0,0 +1,31 @@
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='warehouse_id']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
```
Delete this xpath entry as `warehouse_id` doesn't belong to this view.  Create the following record instead.
```
    <record id="view_order_form_inherit_sale_stock" model="ir.ui.view">
        <field name="name">view.order.form.inherit.sale.stock</field>
        <field name="model">sale.order</field>
        <field name="inherit_id" ref="sale_stock.view_order_form_inherit_sale_stock"/>
        <field name="arch" type="xml">
            <xpath expr="//field[@name='warehouse_id']" position="replace"/>
        </field>
    </record>
```

yostashiro commented on sale_view_adj_hls/views/sale_order_views.xml:
```diff
@@ -0,0 +1,31 @@
+            <xpath expr="//field[@name='commitment_date']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath> 
+            <xpath expr="//field[@name='user_id']" position="attributes">
```
Please use `position="replace"` instead.

yostashiro reviewed (changes requested):
Just some points that I noticed, including the ones we talked on Slack.

nakometal committed:
[FIX] depends

sale -> sales_stock
https://github.com/qrtl/hls-custom/pull/21#pullrequestreview-275276594

nakometal committed:
[FIX] warehouse_id inherit form

https://github.com/qrtl/hls-custom/pull/21#discussion_r314188156

nakometal committed:
[FIX] replace field

https://github.com/qrtl/hls-custom/pull/21#discussion_r314188557

yostashiro committed:
adjust styling

yostashiro committed:
adjust styling
